### PR TITLE
Dom app styling

### PR DESCRIPTION
**Pull request by domeccleston**
## What does this PR do
Restyles appointment cards to match the styling of the coach cards in marketplace

## Description of task to be completed
Reformat appointment cards

## How should this manually be tested
Login to the dashboard and look at the cards

### CODE REVIEW

File: src/views/UserDashboard/UserDashboard.js [hunk 1]
```diff
@@ -126,36 +127,20 @@ const UserDashboard = props => {
...
-              }
-            />
-          ))}
-          <div className='pagination'>
-            <Pagination
-              defaultCurrent={1}
-              defaultPageSize={6}
-              onChange={handlePagination}
-              total={appointments.length}
-            />
```
Comment (BenjaminGrabow) [inline src/views/UserDashboard/UserDashboard.js]: I recognized right now when checking the dev branch, the pagination got also deleted.

Comment (domeccleston) [reply src/views/UserDashboard/UserDashboard.js]: Good points! Editing now

**Comment by domeccleston** [comment]
Have made requested changes in latest commit :)

**Comment by BenjaminGrabow** [comment]
[As a user I want to see appointments](https://trello.com/c/5pt5od4n/102-as-a-user-i-want-to-see-appointments)

**Pull request merged**